src/coverlet.console/coverlet.console.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
+    <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="2.3.4" />
```

Comment:
`Microsoft.Extensions.CommandLineUtils` is a stale dependency. The community fork supports enum parsing which was the reason why I upgraded the dependency.